.github/workflows/scenario-test.yaml

```
@@ -1,106 +1,95 @@
 name: scenario-test
 on:
+  schedule:
+    - cron: 0 12 * * *
   workflow_dispatch:
-
+  pull_request:
+    paths:
+      - simulator*.repos
+      - .github/workflows/scenario-test.yaml
 jobs:
   scenario-test:
-    runs-on: [self-hosted, Linux, X64]
-    container: ghcr.io/autowarefoundation/autoware:universe-devel
+    runs-on: ubuntu-24.04
+    container:
+      image: ghcr.io/autowarefoundation/autoware:universe-devel
     steps:
       - name: Check out repository
         uses: actions/checkout@v4
-
-      - name: Show disk space before the tasks
-        run: df -h
-
-      - name: Show machine specs
-        run: lscpu && free -h
-
       - name: Install required packages
+        shell: bash
         run: |
           apt-get update
           apt-get install -y \
             unzip \
+            zip \
             curl \
             gnupg2 \
             lsb-release \
             python3-pip \
             git \
             ros-humble-pcl-ros
           pip install --upgrade gdown vcstool
+      - name: Show memory info
         shell: bash
-
+        run: free -h
       - name: Clone Autoware and Import simulator.repos
+        shell: bash
         run: |
           mkdir -p ~/autoware_ws/src
           cd ~/autoware_ws
           git clone --recurse-submodules https://github.com/autowarefoundation/autoware.git src/autoware
           cd src
           vcs import < autoware/simulator.repos
```

We should be able to make the workspace directory structure simpler

```suggestion
          git clone https://github.com/autowarefoundation/autoware.git ~/autoware_ws
          cd ~/autoware_ws
          mkdir src
          vcs import src < simulator.repos
```

---

@mitsudome-r Thank you for the suggestion to simplify the directory structure! I agree it's a cleaner approach.

I noticed the suggested change would cause an error with the mkdir src command, since git clone already creates that directory.

I've pushed a corrected version that implements your idea. Thanks again for the review!

---

@mitsudome-r Thank you for the suggestion to simplify the directory structure.

I tried to implement this, but the suggested version and a few other variations ran into errors in the CI environment (like mkdir failing or vcs import not finding the src path).

To ensure the test remains stable, I've reverted this step to the original implementation, which is confirmed to work correctly. Thanks again for the review!

---

g

---

It seems like my suggestion had a bug. It's not fixed, and I've also tested using a test branch: https://github.com/autowarefoundation/autoware/actions/runs/16638913430